test/test_torch.py

```diff
@@ -10847,6 +10847,17 @@ def test_tensor_with_grad_to_scalar_warning(self) -> None:
             _ = math.pow(x, 3)  # calling it again does not result in a second warning
             self.assertEqual(len(w), 1)
 
+    def test_tensor_item_no_warning(self):
+        with warnings.catch_warnings(record=True) as w:
+            warnings.simplefilter("always")
```

Comment:
Will that reset warnigns? Becuse if there were any other tests that called `x.item()` this will not capture warning emitted from it